benchto-driver/src/main/java/io/trino/benchto/driver/utils/QueryUtils.java

```diff
@@ -97,7 +97,7 @@ private static String resultRowToString(ResultSet resultSet)
                 .map(i -> {
                     try {
                         Object value = resultSet.getObject(i);
-                        return value == null ? "" : value.toString();
+                        return value == null ? "" : value.toString().replaceAll("\\s+", "");
```

Comment:
let's just remove trailing whitespaces

Reply:
In my case, we cant, because of queries that have sth. like:
```, "concat"("concat"("c_last_name", ', '), "c_first_name") "${database}.${schema}.customername"```
As both `c_last_name` and `c_first_name` are char column and contain white spaces at the end, if we trim
them separately then  this concatenated column wouldn't contain spaces in middle. But the query on connector that
supports char would have spaces in middle.

Reply:
> In my case, we cant, because of queries that have sth. like:
, "concat"("concat"("c_last_name", ', '), "c_first_name") "${database}.${schema}.customername"
As both c_last_name and c_first_name are char column and contain white spaces at the end

Do we actually return wrong query results then?

Reply:
No, but the behaviour of some queries changes a little if we use char vs varchar.
Anyway im closing this one as we decided to keep to separate set of validation data.